lib/components/dashboard/degree-info/CourseBar.tsx

```diff
@@ -62,16 +57,12 @@ const CourseBar: FC<{
   const openCartPopup = async () => {
     dispatch(updateCartInvokedBySemester(false));
     // Filter for the correst distributions from redux store
-    let distr: UserDistribution = distribution;
-    // get fineReqs
-    if (distribution._id) {
-      distr = await getDistribution(distribution._id, token);
-    }
-    if (distr) {
```

Comment:
You do need this code actually, to display fine requirements for selected distribution!

Reply:
reverted the file